gslb/nodes/avi_model_nodes.go

```diff
@@ -81,26 +81,31 @@ type AviGSK8sObj struct {
 	IPAddr    string
 	Weight    int32
 	// Port and protocol will be only used by LB service
-	Port  int32
-	Proto string
-	TLS   bool
-	Paths []string
+	Port               int32
+	Proto              string
+	TLS                bool
+	Paths              []string
+	VirtualServiceUUID string
+	ControllerUUID     string
+	SyncType           int
```

Comment:
Don't see this being used in rest layer. If we are planning to use this later, we can keep this, else can this be removed ?

Reply:
Thanks, removed it.